index.bs

```diff
@@ -1503,32 +1505,20 @@ these steps:
 
 </div>
 
-# Accessing the Sandboxed File System # {#sandboxed-filesystem}
+# Accessing the Origin Private File System # {#sandboxed-filesystem}
 
-## Sandboxed File System Concepts ## {#sandboxed-filesystem-concepts}
+The <dfn>origin private file system</dfn> is a [=storage endpoint=] whose
+<a spec=storage for="storage endpoint">identifier</a> is `"fileSystem"`,
```

Comment:
Can this just use the `[=storage  endpoint/identifier=]` syntax? Is the `spec=storage` necessary?

Reply:
I guess it would be if there's another spec defining storage endpoint at the moment? Which one?

Reply:
The spec=storage lets me link to it despite it not being exported...

Reply:
Thanks! (I add things to the `anchors` block in that case, whatever you prefer)

Reply:
The downside of the anchors block is that you no longer get an error if the thing you're trying to link to doesn't exist anymore.